lib/isolate_worker.dart

```diff
@@ -53,19 +53,19 @@ abstract class IsolateTask<T, R> {
 
 /// A single worker representing one [Isolate].
 class IsolateWorker<T, R> {
-  ReceivePort receivePort;
-  SendPort _isolatePort;
+  late ReceivePort receivePort;
+  late SendPort _isolatePort;
 
-  Map<int, Completer> _completers = Map();
+  final Map<int, Completer> _completers = <int, Completer>{};
   int get queueLength => _completers.length;
   bool get isBusy => _completers.isNotEmpty;
   static const int MAX_QUEUE = 100;
   bool get isTooBusy => queueLength > MAX_QUEUE;
-  bool _ready = false;
-  StreamSubscription _portSubscription;
+  bool _ready = false; // TODO(filiph): This should be modified somewhere.
```

Comment:
This is the cause of the CI failure. I believe that this flag should be modified somewhere, but isn't. Yay for lint checks, eh?

Reply:
Maybe we can just get rid of it?

Reply:
```suggestion
```